### ISSUE TITLE
Sign in Page not Working

### ISSUE THREAD

**Issue by David310857** [snehitvaddi/ThePotholeReporter-Flutter-App#2]
The Google sign in page not working. Can you review on this?

**Comment by snehitvaddi** [comment]
Hey @David310857!!
Are you on the updated version??
Have a look at this version [Version V1](https://github.com/snehitvaddi/ThePotholeReporter-Flutter-App/releases)

I just checked it on a new device, It is working fine for me.

**Comment by David310857** [comment]
> Hey @David310857!!
> Are you on the updated version??
> Have a look at this version [Version V1](https://github.com/snehitvaddi/ThePotholeReporter-Flutter-App/releases)
> 
> I just checked it on a new device, It is working fine for me.

I had tried the Version V1 still encountering the same problem here. I had tried use different device too still cannot access and no errors in the code. Why still this happen?

**Comment by snehitvaddi** [comment]
Having a look at debugging console could be helpful.
Can you share any errors/issues that were raised in the terminal or Debugging console of Android Studio?

**Comment by David310857** [comment]
![image](https://user-images.githubusercontent.com/77140361/112993135-73947680-919b-11eb-8690-02285d35aa07.png)
![image](https://user-images.githubusercontent.com/77140361/112993419-c79f5b00-919b-11eb-9c29-d78fa7afbc99.png)

**Comment by snehitvaddi** [comment]
Did you check with the updated files?
Is this still persistent?

**Comment by David310857** [comment]
Yeah, I have checked and updated all the files. It is still persistent.
